packages/extension/src/components/pages/Settings/Settings.tsx

```diff
@@ -30,6 +33,21 @@ export const Settings: FC = () => {
 
   const handleLock = useCallback(() => {
     signOut();
+
+    chrome.runtime
```

Comment:
I am wondering if this logic shouldn't be moved to: 
https://github.com/GemWallet/gemwallet-extension/blob/master/packages/extension/src/chromeServices/background/utils/session.ts#L34
This would also handle the timeout of the session that way.

Reply:
done